### PR TITLE
Basic layout and open/export images

### PR DESCRIPTION
**Pull request by Cari1111**
## Changes

- Images can be imported with a button and are copied into a cache folder
- Information to images is saved in `self.images` dictonary
- A preview of the open images are displayed on the left
- Images can be exported (moved to an other directory from the cache folder
- Options for the diffrent models on the right

## Has to be done in the future

- #11
- #12
- #10 To make it more readable

### CODE REVIEW

File: safe_video/ui/app.py [hunk 1]
```diff
@@ -1,37 +1,111 @@
...
+        file_picker_open = ft.FilePicker(on_result=self.upload_callback)
+        file_picker_export = ft.FilePicker(on_result=self.export_callback)
+        page.overlay.append(file_picker_open)
+        page.overlay.append(file_picker_export)
         page.add(
+            ft.Container(ft.Row([
+                ft.Container(content=ft.IconButton(ft.icons.BLUR_ON, focus_color=self.colors.dark), width=50),
+                ft.ElevatedButton("Open Image", on_click=lambda _: file_picker_open.pick_files(file_type=ft.FilePickerFileType.IMAGE, allow_multiple=True), icon=ft.icons.FOLDER_OPEN),
+                ft.ElevatedButton("Export file", on_click=lambda _: file_picker_export.save_file(file_name=self.images[self.current_image_key].name), icon=ft.icons.SAVE_ALT),
+                ft.ElevatedButton("Blur all", on_click=lambda _: self.blur_callback(), icon=ft.icons.PLAY_ARROW),
```
Comment (Kyotoo-chan) [inline safe_video/ui/app.py:83]: This needs to be changed, that the right function is being called.
```suggestion
                ft.ElevatedButton("Blur all", on_click=lambda _: self.settings_callback(), icon=ft.icons.PLAY_ARROW),
```

**Review by Kyotoo-chan — CHANGES_REQUESTED** [review]
Besides that one line, i think the layout is great and I couldnt break it nor find any bugs.

**Review by Kyotoo-chan — APPROVED** [review]

File: safe_video/ui/components.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class PreviewImage(ft.Container):
+    def __init__(self, key, path, callback):
+        super().__init__(
+            key=key,
+            image_src=path,
+            width=50,
+            height=50,
+            on_click=callback,
+            image_fit=ft.ImageFit.FILL,
```
Comment (TheDying0fLight) [inline safe_video/ui/components.py:13]:
```suggestion
            image_fit=ft.ImageFit.COVER,
```

File: safe_video/ui/components.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            border_radius=3,
```
Comment (TheDying0fLight) [inline safe_video/ui/components.py:14]:
```suggestion
            border_radius=10,
```

File: safe_video/ui/components.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    def __init__(self, key, path, callback):
+        super().__init__(
+            key=key,
+            image_src=path,
+            width=50,
+            height=50,
+            on_click=callback,
+            image_fit=ft.ImageFit.FILL,
+            border_radius=3, 
+            image_opacity=0.8
```
Comment (TheDying0fLight) [inline safe_video/ui/components.py:15]:
```suggestion
            image_opacity=1
```

File: safe_video/ui/app.py [hunk 5]
```diff
@@ -1,37 +1,111 @@
...
+
+    def settings_callback(self, info: ft.ControlEvent):
+        print('TODO: Settings')
 
     def build_page(self, page: ft.Page):
         self.page = page
-        Mypick = ft.FilePicker(on_result=self.upload_callback)
-        page.overlay.append(Mypick)
+        page.padding=0
+        page.spacing=0
```
Comment (TheDying0fLight) [inline safe_video/ui/app.py:72]:
```suggestion
        page.padding = 0
        page.spacing = 0
```

**Review by TheDying0fLight — APPROVED** [review]
LGTM

**Pull request merged**